build-support/bin/release_helper.py

```diff
@@ -784,6 +785,29 @@ def upload_wheels_via_twine() -> None:
     )
 
 
+def prompt_to_generate_docs() -> None:
+    has_docs_access = input(
+        "\nThe docs now need to be regenerated. Do you already have editing access to "
+        "readme.com? [Y/n]: "
+    )
+    api_key_url = "https://dash.readme.com/project/pants/v2.5/api-key"
```

Comment:
Does it matter that this includes the version?

Reply:
It should not matter, so long as we do not delete `v2.5`. Readme.com errors if you leave off the version, so I chose an arbitrary one to pin to.